contracts/hydro/src/msg.rs

```diff
@@ -298,10 +298,17 @@ pub struct LockTokensProof {
     pub sig_info: Option<SignatureInfo>,
 }
 
+#[cw_serde]
+pub struct ConvertLockupPayload {
+    pub lock_id: u64,
+    pub sender: String,
```

Comment:
Can we use `Addr` instead of String in order to avoid unnecessary conversions to and from String?